app/models/amenity.rb

```diff
@@ -1,4 +1,6 @@
 class Amenity < ActiveRecord::Base
   has_many :property_amenities
   has_many :properties, through: :property_amenities
+
+  validates :name, presence: true
```

Comment:
cool! I see that you added the validations like stated in your description/PR title.